### ISSUE TITLE
manpage should contain the example from tldr

### ISSUE THREAD

**Issue by sylvestre** [uutils/coreutils#5094]
or at least an option to enable them.
For example, we have it here:
https://uutils.github.io/user/utils/basename.html

But we don't have it here:
https://manpages.debian.org/unstable/rust-coreutils/rust-basename.1.en.html

**Comment by simon04** [comment]
Duplicate of #4845?

**Comment by tertsdiepraam** [comment]
Nope, #4845 is about making the examples better. This one is about all examples missing from the manpages (even though they are present in the online docs).

**Comment by robin-thoene** [comment]
Hey I would try to solve this. To me it looks like the code that puts the examples from the tldr into the online docs lives in `src/bin/uudoc.rs` and smth similar has to happen in `src/bin/coreutils.rs`, `gen_manpage`, right?
